### ISSUE TITLE
Hair displacement, preauricular, towards lateral cheekbone

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4741]
simulconsult

**Comment by pnrobinson** [comment]
This is a feature of Treacher Collins syndrome
https://www.ncbi.nlm.nih.gov/books/NBK1532/

**Comment by pnrobinson** [comment]
This term already exists:
"Projection of scalp hair onto lateral cheek"
https://hpo.jax.org/app/browse/term/HP:0009554
It seems that "Preauricular hair displacement" might be clearer as a primary label.

**Comment by pnrobinson** [comment]
@MickeySegal -- this is HP:0009554
